test/header/CMakeLists.txt

```diff
@@ -19,11 +19,11 @@ option (SEQAN3_FULL_HEADER_TEST "Test seqan3 headers as well as the headers of e
 # header guards are working by including the same header twice.
 #
 # example invocation:
-#     seqan3_header_test (seqan3 "<path>/include/seqan3")
+#     seqan3_header_test (seqan3 "<path>/include/seqan3" "/exclude_regex|more_exclude/")
```

Comment:
I don't understand the example. 
Should more_exclude mean I can add more exclude patterns?
Then I suggest:
```suggestion
#     seqan3_header_test (seqan3 "<path>/include/seqan3" "")  - Nothing will be excluded
#     seqan3_header_test (seqan3 "<path>/include/seqan3" "/exclude_regex/[|more_excludes...]")  - separate multiple exludes with |
```
or can I only specify one more exlude?

Reply:
no you can give any regex

Reply:
I added more info, can you recheck?